programs/server/Server.cpp

```diff
@@ -777,10 +777,21 @@ try
         server_settings.max_backups_io_thread_pool_free_size,
         server_settings.backups_io_thread_pool_queue_size);
 
+    ActivePartsLoadingThreadPool::initialize(
+        server_settings.max_active_parts_loading_thread_pool_size,
+        0, // We don't need any threads once all the parts will be loaded
+        server_settings.max_active_parts_loading_thread_pool_size);
+
     OutdatedPartsLoadingThreadPool::initialize(
         server_settings.max_outdated_parts_loading_thread_pool_size,
-        0, // We don't need any threads one all the parts will be loaded
-        server_settings.outdated_part_loading_thread_pool_queue_size);
+        /*turbo=*/server_settings.max_active_parts_loading_thread_pool_size,
+        0, // We don't need any threads once all the parts will be loaded
+        server_settings.max_outdated_parts_loading_thread_pool_size);
+
+    PartsCleaningThreadPool::initialize(
+        server_settings.max_parts_cleaning_thread_pool_size,
+        0, // We don't need any threads one all the parts will be deleted
+        server_settings.max_parts_cleaning_thread_pool_size);
```

Comment:
I think it could be really good to have a way to configure their sizes on the fly. And ThreadPools already can do that safely. We need just a couple of lines in ConfigReloader. You don't even need any other manual turboMode() with this.

Reply:
Aha, turboMode is not actually manual. Now it totally makes sense for me. But, dynamic changing of size on the fly is also a good thing.

Reply:
Sure, will add, thanks!